gobblin-cluster/src/main/java/gobblin/cluster/GobblinClusterConfigurationKeys.java

```diff
@@ -33,12 +33,16 @@
   // General Gobblin Cluster application configuration properties.
   public static final String APPLICATION_NAME_OPTION_NAME = "app_name";
   public static final String STANDALONE_CLUSTER_MODE = "standalone_cluster";
+  public static final String STANDALONE_CLUSTER_MODE_KEY = GOBBLIN_CLUSTER_PREFIX + "standaloneMode";
```

Comment:
We already have a config that is set for standalone mode, lets converge them in one. Either use this or that (we can make this choice because its not yet eligible for backward compatibility)